data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/AuthConfig.java

```diff
@@ -27,6 +27,12 @@ public static class SaslAuthConfig {
         @JsonProperty("aws_iam")
         private AwsIamAuthConfig awsIamAuthConfig;
 
+        @JsonProperty("ssl_endpoint_identification_algorithm")
+        private String sslEndpointAlgorithm;
+
+        @JsonProperty("plain_config")
```

Comment:
Why this? The plaintext config above has the exactly same config.

Reply:
Yes , we have incorporated the changes.